bitnami/postgresql-ha/values.yaml

```diff
@@ -215,31 +187,55 @@ postgresql:
   ## ref: https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/
   ##
   tolerations: []
+  ## @param postgresql.topologySpreadConstraints Topology Spread Constraints for pod assignment spread across your cluster among failure-domains. Evaluated as a template
+  ## Ref: https://kubernetes.io/docs/concepts/workloads/pods/pod-topology-spread-constraints/#spread-constraints-for-pods
+  ##
+  topologySpreadConstraints: {}
+  ## @param postgresql.priorityClassName Pod priority class
+  ## Ref: https://kubernetes.io/docs/concepts/configuration/pod-priority-preemption/
+  ##
+  priorityClassName: ""
+  ## @param postgresql.schedulerName Use an alternate scheduler, e.g. "stork".
+  ## ref: https://kubernetes.io/docs/tasks/administer-cluster/configure-multiple-schedulers/
+  ##
+  schedulerName: ""
+  ## @param postgresql.terminationGracePeriodSeconds Seconds PostgreSQL pod needs to terminate gracefully
+  ## ref: https://kubernetes.io/docs/concepts/workloads/pods/pod/#termination-of-pods
+  ##
+  terminationGracePeriodSeconds: ""
+  ## @param postgresql.updateStrategy.rollingUpdate PostgreSQL statefulset rolling update configuration parameters
+  ## @param postgresql.updateStrategy.type Postgresql statefulset strategy type
+  ## ref: https://kubernetes.io/docs/concepts/workloads/controllers/statefulset/#update-strategies
+  ##
+  updateStrategy:
+    type: RollingUpdate
+    rollingUpdate: {}
```

Comment:
if `updateStrategy.type` is set to "OnDelete" the parameter `updateStrategy.rollingUpdate` could cause problems. This approach is better:

https://github.com/bitnami/charts/blob/5af36c98d53a5ad3911e0f4dd7fa5c63bb2b37a9/bitnami/apache/values.yaml#L357-L367